Azure/terraform/azure_deployswxtch.tf

```diff
@@ -0,0 +1,108 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 3.0"
+    }
+  }
+}
+
+provider "azurerm" {
+  features {}
+}
+
+locals {
+  cloudswxtch_image_publisher = "swxtchiollc1614108926893"
+  cloudswxtch_image_offer     = "cloudswxtch-vm-001"
+  cloudswxtch_image_version   = "latest"
+}
+
+data "azurerm_resource_group" "resource_group" {
+  name = var.resource_group
+}
+
+data "azurerm_resource_group" "virtualnetwork_resource_group" {
+  name = var.vnet_resource_group
+}
+
+data "azurerm_virtual_network" "virtual_network" {
+  name                = var.vnet_name
+  resource_group_name = data.azurerm_resource_group.virtualnetwork_resource_group.name
+}
+
+data "azurerm_subnet" "ctrlsubnet" {
+  name                 = var.ctrlsubnet_name
+  resource_group_name  = data.azurerm_resource_group.virtualnetwork_resource_group.name
+  virtual_network_name = data.azurerm_virtual_network.virtual_network.name
+}
+
+data "azurerm_subnet" "datasubnet" {
+  name                 = var.datasubnet_name
+  resource_group_name  = data.azurerm_resource_group.virtualnetwork_resource_group.name
+  virtual_network_name = data.azurerm_virtual_network.virtual_network.name
+}
+
+resource "azurerm_network_interface" "control_network_interface" {
+  count               = var.swxtch_count
+  name                = "${var.swxtch_name}-ctrl-nic-${count.index + 1}"
+  location            = data.azurerm_resource_group.resource_group.location
+  resource_group_name = data.azurerm_resource_group.resource_group.name
+
+  ip_configuration {
+    name                          = "cinternal"
+    subnet_id                     = data.azurerm_subnet.ctrlsubnet.id
+    private_ip_address_allocation = "Dynamic"
+    #private_ip_address = var.controlnic_staticip
+  }
+}
+resource "azurerm_network_interface" "data_network_interface" {
+  count                         = var.swxtch_count
+  name                          = "${var.swxtch_name}-data-nic-${count.index + 1}"
+  location                      = data.azurerm_resource_group.resource_group.location
+  resource_group_name           = data.azurerm_resource_group.resource_group.name
+  enable_accelerated_networking = true
+
+  ip_configuration {
+    name                          = "dinternal"
+    subnet_id                     = data.azurerm_subnet.datasubnet.id
+    private_ip_address_allocation = "Dynamic"
+    #private_ip_address            = var.datanic_staticip

```

Comment:
delete if not needed

Reply:
The [docs](https://docs.swxtch.io/docs/deploy-cloudswxtch-terraform-azure?highlight=terraform#steps-to-deploy-this-terraform-script-are-as-follows) reference how to use a static ip for a swxtch instead of a dynamically allocated IP. If you'd prefer I can remove these entries from the documentation and the example.